src/pages/billing/Project.js

```diff
@@ -66,10 +67,10 @@ const WorkspaceCardHeaders = memoWithName('WorkspaceCardHeaders', ({ needsStatus
 const ExpandedInfoRow = ({ title, details, errorMessage }) => {
   const expandedInfoStyles = {
     row: { display: 'flex', justifyContent: 'flex-start', alignItems: 'flex-start' },
-    title: { fontWeight: 600, width: '20%', padding: '0.5rem 1rem 0 2rem', height: '1rem' },
-    details: { width: '20%', marginTop: '0.5rem', height: '1rem', ...Style.noWrapEllipsis },
```

Comment:
This change was because the managed resource group ID (that we show for the Azure workspaces) was getting truncated. In general, I don't see a reason to limit the width of this element, and I think it makes sense to have the label and value more closely related.

Before:
![image](https://user-images.githubusercontent.com/484484/189415170-767bc480-3c57-4a3b-b9f3-0db73fe5d563.png)

After:
![image](https://user-images.githubusercontent.com/484484/189415259-6f9b14fc-031f-4fbc-baae-8179d7272aa6.png)

Reply:
Nice.